tests/app/conftest.py

```diff
@@ -157,6 +157,14 @@ def sample_service(sample_user):
     return service
 
 
+@pytest.fixture(scope="function")
+def sample_service_with_email_branding(sample_service):
+    sample_service.email_branding = create_email_branding(
+        id=uuid.uuid4(), logo="1ac6f483-3105-4c9e-9017-dd7fb2752c44-nhs-blue_x2.png", name="NHS"
```

Comment:
You could optionally use a non-NHS example for this fixture just in case someone tries to use this along with the `nhs_email_branding` fixture in the same test and gets a surprising integrity error - but that's probably very unlikely to happen.

Reply:
Patched in fixup commit 5da28b6b9ed665cef0dbe809bb83c286ea0c9b43